conf/application.conf

```diff
@@ -126,7 +126,7 @@ google-analytics {
 }
 
 assets {
-  version = "3.8.0"
+  version = "3.3.2"
```

Comment:
Why this change?

`3.3.2` assets has problem with jQuery

Reply:
Shouldn't have committed that change - was done in the "IE panic" changes on Friday

Reply:
Done